include/fixed_containers/optional_reference.hpp

```diff
@@ -43,6 +43,15 @@ class OptionalReference
     {
     }
 
+    constexpr OptionalReference(pointer ptr)
```

Comment:
ctors which don't raise exceptions should be marked `noexcept`.

Reply:
Done.